rspec-mode.el

```diff
@@ -321,7 +321,8 @@ buffers concurrently"
               (yas-activate-extra-mode 'rspec-mode)
             (make-local-variable 'yas-extra-modes)
             (add-to-list 'yas-extra-modes 'rspec-mode)
-            (yas--load-pending-jits))))
```

Comment:
It's better to add `declare-function` statements instead.

So that if/when is private function is renamed or deleted, the user gets an error rather than a silent misbehavior.

Reply:
Thanks for the suggestion. I've changed to use `declare-function` in the latest revision.

Reply:
Thanks!

I've pushed a follow-up cleanup: `declare-function` is really meant to be used at top level (putting it inside a function doesn't change anything), and yasnippet 0.8.1 is really old enough by now.